ansible/deploy_db.yml

```diff
@@ -11,6 +11,7 @@
     - riakcs
     - formplayer
     - pg_standby
+    - airflow
```

Comment:
I don't think having the DB on the same VM is necessary - even if it is this should be accomplished in the inventory file by putting the airflow host in the `postgresql` group